framework/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Yii Framework 2 Change Log
 2.0.16 January 30, 2019
 -----------------------
 
+- Bug #17094: Fixed response on 204 status. Now it is empty (GHopperMSK)
```

Comment:
IMO `2.0.16.1` should have its own changelog section.

Reply:
Sure.